scss/_utilities_content-align.scss

```diff
@@ -58,4 +58,12 @@
       text-align: right !important;
     }
   }
+
+  .u-align-text--small-to-default {
+    padding-top: map-get($nudges, nudge--small) + $sp-unit;
+  }
+
+  .u-align-text--x-small-to-default {
```

Comment:
What are all these undocumented utility classes? They are like a riddle. What is small to default even mean?

Reply:
The goal is to make it possible to align small text to a paragraph. Without it, they are slightly misaligned because of the difference in line-heights (first row)

![image](https://user-images.githubusercontent.com/2741678/108083181-b88c9f80-706a-11eb-87df-186c7b9b9651.png)

Compare with the second row where they are perfectly aligned.

I can change the name to something less cryptic? Just wasn't sure what to call it.